portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,19 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
I think your html doesn't call the function? I think you can add it in this PR since it only add little complexity to the PR.

Reply:
I probably should have taken it out for this PR, but the removal ended up in #2